app/controllers/Account.scala

```diff
@@ -117,7 +117,7 @@ final class Account(
 
   private def doNowPlaying(me: lila.user.User, req: RequestHeader) =
     env.round.proxyRepo.urgentGames(me) map { povs =>
-      val nb = (getInt("nb", req) | 9) atMost 50
```

Comment:
not a fan of this rewrite that assumes all devs know all scala precedence rules perfectly.